lib/sedna/service/server/aggregation.py

```diff
@@ -241,3 +268,17 @@ async def client_info(self, request: Request):
         if client_id:
             return server.get_client(client_id)
         return WSClientInfoList(clients=server.client_list)
+
+import os
+os.environ['config_file'] = '/home/work/server.yml'
```

Comment:
The import module can be add on the top.

Reply:
updated